io-engine/src/subsys/config/opts.rs

```diff
@@ -84,6 +84,100 @@ impl GetOpts for NexusOpts {
 /// Must be equal to the size of `spdk_nvmf_target_opts.crdt`.
 pub const TARGET_CRDT_LEN: usize = 3;
 
+#[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
+pub enum NvmfTgtTransport {
+    Rdma,
+    #[default]
+    Tcp,
+}
```

Comment:
Why not use strum macros?

Reply:
Thanks, this is something new. Added `strum` annotations now.